packages/sdk-react/src/core/queries/useMe.ts

```diff
@@ -0,0 +1,24 @@
+import { useMoniteContext } from '@/core/context/MoniteContext';
+
+export const useMyEntity = () => {
+  const { api } = useMoniteContext();
+
+  const queryProps = api.entityUsers.getEntityUsersMyEntity.useQuery(
+    {},
+    {
+      retry: false,
+      staleTime: Infinity,
+      refetchOnMount: false,
+      refetchOnWindowFocus: false,
+    }
```

Comment:
We usually don’t pass specific configurations for queries. Please refer to other examples, such as in `packages/sdk-react/src/core/queries/useCounterpart.ts`

 Ideally, it should just be:
`const queryProps = api.entityUsers.getEntityUsersMyEntity.useQuery()`

Reply:
That particular section was `cherry-picked`, from the options being passed, I can also understand why because this is a fetch for a user top-level information (entity) which ideally shouldn't change or at least frequently that would require retries, stale time before refetching the data, fetching when focus is shifted in and out of the browser window or mount. 

Except if these options are passed by default, I think it's nice they are added to that specific use-case, but you can advise otherwise why they shouldn't be; I am happy to remove them as well.

Reply:
Justifying them for this specific scenario seems reasonable from your explanation. I would just ask then to add extra comment section on of the function explaning that, because we are introducing a deviation from the default settings 

/**
 * @note We are deviating from the default query configuration because the data 
 * does not require frequent refetching or retries as the data fetched
 * (user entity information) is relatively static
 * 
 * @returns {QueryResult} The result of the user entity fetch.
 */
export const useMyEntity = () => {

Reply:
Comment added.